lib/y-richtext.coffee

```diff
@@ -187,7 +180,10 @@ class YRichText extends BaseClass
     @_model.getContent("characters").observe (events) => @locker.try ()=>
       for event in events
         delta =
-          ops: [{retain: event.position}]
+          ops: []
```

Comment:
we want to send `retain` only when it is necessary. Will prevent some inconsistencies